src/Main/Main.java

```diff
@@ -0,0 +1,186 @@
+package Main;
+
+import Address.*;
+import Book.*;
+import Users.*;
+import Users.Client;
+
+import java.sql.SQLOutput;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Main {
+
+
+    public static void main(String[] args) throws ISBN.ISBNFormatException {
+        //a valid ISBN for testing: 0569934052
+        MainService service = MainService.getInstance();
+        Scanner read = new Scanner(System.in);
+        int option;
+        do {
+            int option1;
+            System.out.println("1 - Librarian");
+            System.out.println("2 - Client");
+            option = read.nextInt();
+            switch (option) {
+                case (1): {
+                    do {
+                        System.out.println("1 - Creare cont nou");
+                        System.out.println("2 - Logare in cont existent");
+                        System.out.println("3 - Add a new book");
+                        System.out.println("4 - Edit a book");
+                        System.out.println("5 - Remove a book");
+                        System.out.println("6 - See the users sorted by email");
+                        System.out.println("7 - Change the password");
+                        System.out.println("8 - Log out");
+                        System.out.println("9 - Add a new author");
+                        System.out.println("10 - Remove an author");
+                        System.out.println("11 - exit");
```

Comment:
I would suggest to stick to romanian or english in your app (it happens to me as well to use rom-english but in a real application this should not happen and since we are exercising now for a real project, it would be nice to have this :D )